weldx/transformations/cs_manager.py

```diff
@@ -1955,7 +1955,7 @@ def _get_lcs(edge):
                 return self.graph.edges[(edge[1], edge[0])]["lcs"]
 
             lcs_list = [_get_lcs(edge) for edge in list_of_edges]
-            lcs_list = [lcs for lcs in lcs_list if lcs.time is not None]
+        lcs_list = [lcs for lcs in lcs_list if lcs.time is not None]
```

Comment:
So this was the line that caused the trouble. It was only included in the branch that was executed if one specified a list of edges. So for the default case, an LCS with a TimeSeries wouldn't be sorted out and cause a `None` value in the time union. It's funny, that the less used case (providing a list of edges) was actually indirectly covered, while the default case wasn't. So it slipped through except for the notebook. I now added a set of dedicated tests 😉